tests/python/unittest/test_gluon_rnn.py

```diff
@@ -371,22 +160,10 @@ def forward(self, inpt):
         net_params[k].set_data(weights[k])
         ref_net_params[k.replace('l0', '_lstm_fwd.l0').replace('r0', '_lstm_bwd.l0')].set_data(weights[k])
 
-    data = mx.random.uniform(shape=(11, 10, in_size))
+    data = mx.np.random.uniform(size=(11, 10, in_size))
     assert_allclose(net(data).asnumpy(), ref_net(data).asnumpy(), rtol=1e-04, atol=1e-02)
 
 
-
-def test_zoneout():
```

Comment:
Should this test be adapted to testing the front-end infer_shape functions instead?